views/js/component/results.js

```diff
@@ -0,0 +1,251 @@
+                var self = this;
+
+                areaBroker.getListArea()
+                    .empty()
+                    .data('ui.datatable', null)
+                    .off('.datatable')
+                    .on('load.datatable', function() {
+                        self.trigger('load');
+                    })
+                    .datatable({
+                        url  : config.dataUrl,
+                        model : config.model,
+                        actions : actions
+                    });
+
+                pluginRun('render')
+                    .then(function () {
+                        self.trigger('render');
+                    })
+                    .catch(function (err) {
+                        self.trigger('error', err);
+                    });
+
+                return this;
+            },
+
+            /**
+             * Refreshes the list
+             * @returns {resultsList} chains
+             */
+            refresh: function refresh() {
+                areaBroker.getListArea().datatable('refresh');
+                return this;
+            },
+
+            /**
+             * Cleans up everything and destroys the component
+             * @returns {resultsList} chains
+             * @fires resultsList#destroy - once everything has been destroyed
+             * @fires resultsList#error - if something went wrong
+             */
+            destroy: function destroy() {
+                var self = this;
+
+                pluginRun('destroy')
+                    .then(function () {
+                        areaBroker.getListArea().empty();
+
+                        self.trigger('destroy');
+                    })
+                    .catch(function (err) {
+                        self.trigger('error', err);
+                    });
+                return this;
+            },
+
+            /**
+             * Add a line action
+             * @param {String|Object} name
+             * @param {Function|Object} [action]
+             * @returns {resultsList} chains
+             * @fires resultsList#addaction - notify the action add
+             */
+            addAction: function addAction(name, action) {
+                var descriptor;
+
+                if (_.isPlainObject(name)) {
+                    descriptor = name;
+                } else if (_.isPlainObject(action)) {
+                    descriptor = action;
+                    action.id = name;
+                } else {
+                    descriptor = {
+                        id: name,
+                        label: name
+                    }
+                }
+
+                if (_.isFunction(action)) {
+                    descriptor.action = action;
+                }
+                if (!descriptor.label) {
+                    descriptor.label = descriptor.id;
+                }
+
+                actions.push(descriptor);
+
+                this.trigger('addaction', descriptor);
+
```

Comment:
Well, it is an API. So this event has been added to be consistent with the overall events. Every actions that have side effects have their own event, why only not this one?
And also this event is used inside the unit tests (I know, this is not a strong argument :) ).